### ISSUE TITLE
[Accessibility] SfTextInputLayout - HelperText and ErrorText not visible for screen readers

### ISSUE THREAD

**Issue by bartlomiejsawarski** [syncfusion/maui-toolkit#181]
### Description

If we use VoiceOver (macOS) or Narrator (Windows) to read through ui controls on the screen we would not be able to reach helper and error texts.

### Steps to Reproduce

1. [Add syncfusion toolkit to empty MAUI project](https://github.com/syncfusion/maui-toolkit?tab=readme-ov-file#configure-syncfusion-toolkit)
2. In `MainPage.xaml` just add below code to `StackLayout`

```
            <textInputLayout:SfTextInputLayout
                Hint="SFTextInputLayout"
                ErrorText="This is error text"
                HasError="True"
                HorizontalOptions="Fill">
                <Entry />
            </textInputLayout:SfTextInputLayout>

            <textInputLayout:SfTextInputLayout Hint="Padding"
                                           InputViewPadding="0,5,0,5"
                                           ContainerType="Outlined"
                                           HelperText="Top and bottom padding is 5">
                <Entry />
            </textInputLayout:SfTextInputLayout>

            <textInputLayout:SfTextInputLayout Hint="Name"
                                           IsHintAlwaysFloated="true"
                                           ContainerType="Outlined"
                                           HelperText="Enter your name">
                <Entry />
            </textInputLayout:SfTextInputLayout>
```
3. Run VoiceOver/Narrator and move between ui controls

https://github.com/user-attachments/assets/851c90f3-4223-4509-b67c-ae9bd299e00a

### Version with bug

1.0.4

### Is this a regression from previous behavior?

Not sure, haven't tested other versions

### Last Known Working Version

1.0.3

### Affected platforms

Windows, macOS

### Have you found a workaround?

No, but I think it's not working because HelperText and ErrorText are not Labels, rather drawings on Canvas

### Relevant log output

```shell

```
